datasets/acute-care/families_to_samples.py

```diff
@@ -98,6 +98,28 @@ def get_family_to_sample_map(
     return dict_result
 
 
+def get_escaped_json_string(pedigree_dict: dict) -> str:
+    """
+    This string is used as an argument to analysis-runner, which submits it as a string argument to a dataproc process
+    By escaping we ensure the second round of arguments are received correctly
+    """
+    json_string = json.dumps(pedigree_dict, separators=(",", ":"))
+    escaped = json_string.translate(
+        str.maketrans(
+            {
+                "{": r"\{",
+                "}": r"\}",
+                "]": r"\]",
+                "[": r"\[",
+                ",": r"\,",
+                ":": r"\:",
+                '"': r'\"',
+            }
+        )
+    )
+    return escaped
```

Comment:
My understanding is a double dump (ie: json.dumps the resulting string) should do the escaping. Or during the print on line 229, why not `shlex.quote` and wrap in single quotes?

Or is the problem that the analysis runner is doing a poor job of quoting arguments passed to it, because we should fix the globally if so.

Reply:
Ahh dangit, I think shlex was what I was looking for but I forgot all about it... That should do the job just fine. 

I don't think this is a typical example, as passing a string-formatted JSON object instead of a pointer to a JSON file is a nightmare. The forwarding of the string argument to a second command line invocation for the dataproc cluster shouldn't cause problems with other types of String